src/packages/client/src/generation/TSClient/Model.ts

```diff
@@ -76,7 +76,7 @@ export class Model implements Generatable {
             this.collector,
           ),
         )
-      } else if (action !== 'groupBy') {
+      } else if (action !== 'groupBy' && action !== 'aggregate') {
```

Comment:
Isn't this unrelated to JSDocs?

Reply:
Without this, the `Aggregate{Model}Args` are added twice. The same reason why you ignored `groupBy`

Reply:
Ok makes sense